cache/cache.go

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
dskit is Apache 2

Reply:
It looks like the rest of dskit just doesn't have a license line at the top and is covered by the LICENSE file - so I think we can go ahead and delete these lines